revdep/README.md

```diff
@@ -1,74 +1,62 @@
-|diffobj      |0.3.5  |NA         |*  |
-|digest       |0.6.29 |0.6.29     |   |
-|ellipsis     |0.3.2  |NA         |*  |
-|evaluate     |0.15   |0.15       |   |
-|fansi        |1.0.3  |NA         |*  |
-|glue         |1.6.2  |1.6.2      |   |
-|highr        |0.9    |0.9        |   |
-|httr         |1.4.3  |NA         |*  |
-|jsonlite     |1.8.0  |1.8.0      |   |
-|knitr        |1.39   |1.39       |   |
-|lazyeval     |0.2.2  |0.2.2      |   |
-|lifecycle    |1.0.1  |NA         |*  |
-|magrittr     |2.0.3  |2.0.3      |   |
-|mime         |0.12   |NA         |*  |
-|openssl      |2.0.2  |NA         |*  |
-|pillar       |1.7.0  |NA         |*  |
-|pkgconfig    |2.0.3  |NA         |*  |
-|pkgload      |1.2.4  |NA         |*  |
-|praise       |1.0.0  |NA         |*  |
-|processx     |3.6.0  |3.6.0      |   |
-|ps           |1.7.0  |1.7.0      |   |
-|R6           |2.5.1  |2.5.1      |   |
-|rematch2     |2.1.2  |NA         |*  |
-|remotes      |2.4.2  |2.4.2      |   |
-|rex          |1.2.1  |1.2.1      |   |
-|rlang        |1.0.2  |NA         |*  |
-|rprojroot    |2.0.3  |2.0.3      |   |
-|rstudioapi   |0.13   |NA         |*  |
-|stringi      |1.7.6  |1.7.6      |   |
-|stringr      |1.4.0  |1.4.0      |   |
-|sys          |3.4    |NA         |*  |
-|testthat     |3.1.4  |NA         |*  |
-|tibble       |3.1.7  |NA         |*  |
-|utf8         |1.2.2  |NA         |*  |
-|vctrs        |0.4.1  |NA         |*  |
-|waldo        |0.4.0  |NA         |*  |
-|withr        |2.5.0  |2.5.0      |   |
-|xfun         |0.31   |0.31       |   |
-|xml2         |1.3.3  |1.3.3      |   |
-|xmlparsedata |1.0.5  |1.0.5      |   |
-|yaml         |2.3.5  |2.3.5      |   |
+|package      |old     |new        |Δ  |
+|:------------|:-------|:----------|:--|
+|lintr        |3.1.0   |3.1.0.9000 |*  |
+|backports    |1.4.1   |1.4.1      |   |
+|callr        |3.7.3   |3.7.3      |   |
+|cli          |3.6.1   |3.6.1      |   |
+|crayon       |1.5.2   |1.5.2      |   |
+|cyclocomp    |1.1.1   |1.1.1      |   |
+|desc         |1.4.2   |1.4.2      |   |
+|digest       |0.6.33  |0.6.33     |   |
+|evaluate     |0.21    |0.21       |   |
+|glue         |1.6.2   |1.6.2      |   |
+|highr        |0.10    |0.10       |   |
+|knitr        |1.44    |1.44       |   |
+|lazyeval     |0.2.2   |0.2.2      |   |
+|processx     |3.8.2   |3.8.2      |   |
+|ps           |1.7.5   |1.7.5      |   |
+|R6           |2.5.1   |2.5.1      |   |
+|remotes      |2.4.2.1 |2.4.2.1    |   |
+|rex          |1.2.1   |1.2.1      |   |
+|rprojroot    |2.0.3   |2.0.3      |   |
+|withr        |2.5.0   |2.5.0      |   |
+|xfun         |0.40    |0.40       |   |
+|xml2         |1.3.5   |1.3.5      |   |
+|xmlparsedata |1.0.5   |1.0.5      |   |
+|yaml         |2.3.7   |2.3.7      |   |
 
 # Revdeps
 
+## Failed to check (11)
+
+|package             |version |error |warning |note |
+|:-------------------|:-------|:-----|:-------|:----|
+|admiralophtha       |0.2.0   |1     |        |     |
```

Comment:
also failing on CRAN.

Reply:
HI,  I'm the maintainer of admiralophtha. May I ask - is this failing with admiralophtha's 0.3.0 version - just released on CRAN this week? I can't detect anything wrong over there.